packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testFindLevelOfHeadingLevels1To6.st

```diff
@@ -0,0 +1,5 @@
+tests
+testFindLevelOfHeadingLevels1To6
+	
+	#('#' 1 '##' 2 '###' 3 '####' 4 '#####' 5 '######' 6) pairsDo: [:string :level | 
```

Comment:
Here I would prefer an `Dictionary`, because you map input strings to expected heading levels. Also, the `->` would help to draw the boundaries between the cases.

Also applies to `testTextAttributesOfLevels1To6`